programs/drift/src/state/high_leverage_mode_config.rs

```diff
@@ -0,0 +1,33 @@
+use crate::error::DriftResult;
+use crate::error::ErrorCode;
+use crate::state::traits::Size;
+use crate::validate;
+use anchor_lang::prelude::*;
+
+#[account(zero_copy(unsafe))]
+#[derive(Default, Eq, PartialEq, Debug)]
+#[repr(C)]
+pub struct HighLeverageModeConfig {
+    pub max_users: u32,
+    pub current_users: u32,
+    pub padding: [u8; 32],
```

Comment:
maybe there should be a reduce only mode, so can allow number go down naturally to smaller set over time